### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by MacCrazyman**
this branch contains:

* Basic structure of the "to-do list" project.
* Basic style for the interface.
* Tasks list created dynamically using Javascript.
* Webpack used to generate distribution files.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  {
+    description: 'today\'s task',
+    completed: 0,
+    index: 0,
+  },
+  {
+    description: 'yesterday\'s tast description',
+    completed: 0,
+    index: 0,
+  },
```
Comment (DeliceLydia) [inline src/index.js:20]: - [x] On page load, the list array should be rendered dynamically to the DOM please work on that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @MacCrazyman, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (MacCrazyman) [reply src/index.js:20]: Hello @DeliceLydia , thanks for your feedback
It looks that I broke the code when I was fixing the linters errors in commit 9658a201f6ae313a2c41c1a189f149f980c4c8f6
https://github.com/MacCrazyman/to-do-list/blob/9658a201f6ae313a2c41c1a189f149f980c4c8f6/src/index.js#L24
https://github.com/MacCrazyman/to-do-list/blob/85de91fd54caad9c2b6a128693eaa63ef7c74c79/src/index.js#L24

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @MacCrazyman, 

You have solved the problem with the loading of the list array.  :+1: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


## Optional suggestions
- [x] Please update your Readme file with descriptive title and description.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**